data/mods/Xedra_Evolved/effects/effects.json

```diff
@@ -112,6 +106,19 @@
     ],
     "rating": "good"
   },
+  {
+    "type": "effect_type",
+    "id": "dreamer_eff",
+    "name": [ "Matter Twist", "Matter Deformation", "Dimensional Shift", "Dimensional Chasm", "Reality Gateway" ],
+    "desc": [
+      "Sometimes the world nearby twist.  Was it you?",
+      "Sometimes there is a odd noise you hear.  Sound like whisper.",
+      "You feel someone's presences.  They are wondering.",
```

Comment:
Are they wondering like thinking and you can kind of hear those thoughts or are they wandering around nearby?  Wanted to make sure I understand your intention on this.

Reply:
More like "you feel they are interested in you, but still mostly neutral"